src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -0,0 +1,343 @@
+package org.broadinstitute.hellbender.tools.genomicsdb;
+
+import com.intel.genomicsdb.ChromosomeInterval;
+import com.intel.genomicsdb.GenomicsDBImporter;
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.tribble.AbstractFeatureReader;
+import htsjdk.tribble.FeatureReader;
+import htsjdk.tribble.readers.LineIterator;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.vcf.VCFCodec;
+import htsjdk.variant.vcf.VCFHeader;
+import htsjdk.variant.vcf.VCFHeaderLine;
+import htsjdk.variant.vcf.VCFUtils;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.VariantProgramGroup;
+import org.broadinstitute.hellbender.engine.GATKTool;
+import org.broadinstitute.hellbender.engine.ProgressMeter;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.broadinstitute.hellbender.utils.nio.SeekableByteChannelPrefetcher;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.channels.SeekableByteChannel;
+import java.nio.file.Files;
+import java.util.*;
+import java.util.function.Function;
+
+/**
+ * This tool imports GVCFs to GenomicsDB. To run this tool,
+ * 1. A single interval must be provided
+ * 2. The tool accepts multiple GVCFs each of which must contain data
+ *    for one sample
+ * 3. The path to the GenomicsDB workspace must be specified
+ * 4. User may optionally specify paths to which to write JSON files
+ *
+ * To read data from GenomicsDB, use the query interface GenomicsDBFeatureReader
+ */
+@CommandLineProgramProperties(
+    summary = "Import VCFs to GenomicsDB",
+    oneLineSummary = "Import VCFs to GenomicsDB",
+    programGroup = VariantProgramGroup.class
+)
+public final class GenomicsDBImport extends GATKTool {
+
+    private final long DEFAULT_VCF_BUFFER_SIZE_PER_SAMPLE = 16*1024L;
+    private final long DEFAULT_SEGMENT_SIZE = 1048576L;
+    private final String DEFAULT_ARRAY = "genomicsdb_array";
+    private final String DEFAULT_VIDMAP_FILE_NAME = "vidmap.json";
+    private final String DEFAULT_CALLSETMAP_FILE_NAME = "callset.json";
+
+    private static final String WORKSPACE_ARG_NAME = "GenomicsDBWorkspace";
```

Comment:
Can you make the first letter of each of your arguments lowercase, following the standard "camelCase" convention?